backend/src/v2/cmd/compiler/main.go

```diff
@@ -35,18 +43,34 @@ var (
 	launcher     = flag.String("launcher", "", "v2 launcher image")
 	driver       = flag.String("driver", "", "v2 driver image")
 	pipelineRoot = flag.String("pipeline_root", "", "pipeline root")
+	httpProxy    = flag.String(httpProxyArg, unsetProxyArgValue, "The proxy for HTTP connections.")
+	httpsProxy   = flag.String(httpsProxyArg, unsetProxyArgValue, "The proxy for HTTPS connections.")
+	noProxy      = flag.String(noProxyArg, unsetProxyArgValue, "Addresses that should ignore the proxy.")
 )
 
 func main() {
```

Comment:
I couldn't find where this command is used. If it's user facing only (i.e. backend doesn't call it) then I think we should not require the proxy flags as it impacts UX for the majority of cases.

Reply:
It's the DAG driver entry point. It's not user-facing. I tried to make it mandatory only for non-user-facing parts.
The only user-facing part should be the apiserver env vars. If you find any other, please let me know.